### DISCUSSION TITLE
only index.html works with cleanUrls

### DISCUSSION THREAD

**Discussion by lctrt** [vercel/community#403] [Help] [answered]
custom static website has this structure:
<img width="246" alt="image" src="https://user-images.githubusercontent.com/4627980/157118627-287cb56a-fa58-404e-9d0c-d42759b95c6a.png">

index.html works fine with .com/index but others don't. .com/about should go to about.html but it gives a 404 not found page instead. Custom 404 pages doesn't work either. 

Codebase that deployed fine 3 days ago is not working anymore, even with identical code.

**Comment by gogocharli** [comment, marked as answer]
Hello @lctrt, thank you for raising this issue. It should now be fixed after redeploying your project.

**Comment by lctrt** [comment]
@gogocharli  looking good, thank you!
